pyproject.toml

```diff
@@ -7,3 +7,33 @@ target_version = ["py36"]
 
 [tool.setuptools_scm]
 local_scheme = "no-local-version"
+
+[tool.isort]
+profile = "black"
+known_third_party = ["humanize"]
+
+[tool.pytest.ini_options]
+addopts = "--color=yes --doctest-modules"
```

Comment:
```suggestion
[tool.pytest.ini_options]
addopts = "--strict-config --strict-markers --color=yes --doctest-modules"
xfail_strict = true
junit_family = "xunit2"
filterwarnings = ["error"]
```

Reply:
@hugovk do you want `--color=yes` enabled? it was in setup.cfg where it was ignored in favour of the config in `tox.ini`

Reply:
As long as we get coloured pytest output on GitHub Actions, it's fine to keep it in just one place.

Although please could we keep the tox.ini file? I'd prefer to keep that stuff in there because it's easier to look for and more common to have the test commands in there.

(pytest config can come here.)

Thanks!